htdocs/tutorials/getting-started.html

```diff
@@ -93,7 +93,7 @@
 	</li>
 </ul>
 			<div id="content">
-				<h1>Getting started</h1><p>Getting started with GoldenLayout is a breeze, but a bit off handholding is always nice. This tutorial takes you through the initial steps until you&#39;re up and running with your first managed layout.</p><h3>Including dependencies</h3><p>GoldenLayout needs jQuery to work - that’s it. Either include it directly, using a script tag, e.g.</p><pre><code><span class="token operator" >&lt;</span>script type<span class="token operator" >=</span><span class="token string" >"text/javascript"</span> src<span class="token operator" >=</span><span class="token string" >"http://code.jquery.com/jquery-1.11.1.min.js"</span><span class="token operator" >></span>
+				<h1>Getting started</h1><p>Getting started with GoldenLayout is a breeze, but a bit of handholding is always nice. This tutorial takes you through the initial steps until you&#39;re up and running with your first managed layout.</p><h3>Including dependencies</h3><p>GoldenLayout needs jQuery to work - that’s it. Either include it directly, using a script tag, e.g.</p><pre><code><span class="token operator" >&lt;</span>script type<span class="token operator" >=</span><span class="token string" >"text/javascript"</span> src<span class="token operator" >=</span><span class="token string" >"http://code.jquery.com/jquery-1.11.1.min.js"</span><span class="token operator" >></span>
```

Comment:
Hi! I'm being blind: where's the typo? Can you also change the source file https://github.com/deepstreamIO/golden-layout-website/blob/d899dba2f9eb0d4f0a1f7e6f0bec6e3e5d2b72df/pages/tutorials/getting-started.md and/or point out the typo so I can do it :)